src/sssom_schema/schema/sssom_schema.yaml

```diff
@@ -59,22 +63,36 @@ enums:
       rdf property:
         meaning: rdf:Property
       composed entity expression:
+        meaning: sssom:ComposedEntityExpression
```

Comment:
Composed entity expressions to model coordinated concepts are an ugly abnormity anyway, so I don't mind. In JSKOS there is no special type for composed entities but [concept bundles](https://gbv.github.io/jskos/#concept-bundles) are used for several purposes. Allowing slot `subject_id` and `object_id` to have multiple values might better solve this but nevermind.